threatfox/package.yaml

```diff
@@ -64,17 +64,20 @@ pipelines:
       A pipeline that periodically fetches the ThreatFox IOC database and
       publishes the feed to the `threatfox` topic.
     definition: |
-      every {{ inputs.refresh-interval }}
-        from https://threatfox-api.abuse.ch/api/v1/
-          query=get_iocs
-          days:={{ inputs.days }}
```

Comment:
Is this translation correct? httpie docs mention this is raw json field and I'm not sure this goes into header?

Reply:
This was just wrong before.